spikes/lk_behavior_kit/main.cpp

```diff
@@ -22,51 +22,128 @@
 using namespace leka;
 using namespace std::chrono;
 
-auto sd_blockdevice = SDBlockDevice {SD_SPI_MOSI, SD_SPI_MISO, SD_SPI_SCK};
-auto fatfs			= FATFileSystem {"fs"};
+//
+// MARK: - Global definitions
+//
 
-auto videokit = VideoKit {};
+namespace {
 
-auto corespi_belt		   = CoreSPI {LED_BELT_SPI_MOSI, NC, LED_BELT_SPI_SCK};
-auto corespi_ears		   = CoreSPI {LED_EARS_SPI_MOSI, NC, LED_EARS_SPI_SCK};
-auto ears				   = CoreLED<LedKit::kNumberOfLedsEars> {corespi_ears};
-auto belt				   = CoreLED<LedKit::kNumberOfLedsBelt> {corespi_belt};
-auto animation_thread	   = rtos::Thread {};
-auto animation_event_queue = events::EventQueue {};
+namespace sd {
 
-auto ledkit = LedKit {animation_thread, animation_event_queue, ears, belt};
+	namespace internal {
 
-auto motor_left_dir_1  = mbed::DigitalOut {MOTOR_LEFT_DIRECTION_1};
-auto motor_left_dir_2  = mbed::DigitalOut {MOTOR_LEFT_DIRECTION_2};
-auto motor_left_speed  = CorePwm {MOTOR_LEFT_PWM};
-auto motor_right_dir_1 = mbed::DigitalOut {MOTOR_RIGHT_DIRECTION_1};
-auto motor_right_dir_2 = mbed::DigitalOut {MOTOR_RIGHT_DIRECTION_2};
-auto motor_right_speed = CorePwm {MOTOR_RIGHT_PWM};
+		auto bd = SDBlockDevice {SD_SPI_MOSI, SD_SPI_MISO, SD_SPI_SCK};
+		auto fs = FATFileSystem {"fs"};
```

Comment:
Not a big fan of acronyms, except for the well-known like WiFi or SD, we should use an explicit name or use a glossary